Resources/Locale/en-US/reagents/meta/biological.ftl

```diff
@@ -30,3 +30,6 @@ reagent-desc-vomit = You can see a few chunks of someone's last meal in it.
 
 reagent-name-grey-matter = grey matter
 reagent-desc-grey-matter = Thought juice, the stuff that leaks out of your ears.
+
+reagent-name-coffeegrounds = coffee grounds
+reagent-desc-coffeegrounds = Fresh coffee grounds. Aromatic, and richly textured, exuding a robust, earthy scent that promises a flavorful brew.
```

Comment:
Consider moving to Resources/Locale/en-us/_NF/reagents/meta/biological.ftl to avoid merge conflicts from upstream changes.